### PR TITLE
support lbzip2

### PR DESCRIPTION
**Pull request by missinglink**
This PR contains https://github.com/pelias/whosonfirst/pull/481 with another commit to tidy up the formatting/indentation.
See https://github.com/pelias/whosonfirst/issues/480 & https://github.com/pelias/whosonfirst/pull/481 for more information.
Credit for the work goes to @loadit1 and the authorship in git has been maintained for posterity.

I've also prefixed the latest commit with `feat(xxx):` which will result in a minor version bump.

Closes https://github.com/pelias/whosonfirst/pull/481
Resolves https://github.com/pelias/whosonfirst/issues/480

### CODE REVIEW

File: utils/download_sqlite_all.js [hunk 1]
```diff
@@ -50,9 +51,19 @@ function download(callback) {
   const generateCommand = (sqlite, directory) => {
     let extract;
     if (/\.db\.bz2$/.test(sqlite.name_compressed)) {
-      extract = `bunzip2`;
-    } else if(/\.db\.tar\.bz2$/.test(sqlite.name_compressed)) {
-      extract = `tar -xjO`;
+      // Check if we have lbzip2 installed
+      if (commandExistsSync('lbzip2')) {
+        extract = 'lbzip';
```
Comment (orangejulius) [inline utils/download_sqlite_all.js]: When testing this code I ran into an issue with this command. Shouldn't it be `lbzip2` here?

File: utils/download_sqlite_all.js [hunk 2]
```diff
@@ -50,9 +51,19 @@ function download(callback) {
...
+      } else {
+        extract = 'bunzip';
```
Comment (loadit1) [inline utils/download_sqlite_all.js]: according tar man page, flag `-j` is for `bzip2` not `bzip`. I belive that this is just typo and here we should has `bunzip2` command as it was on my original PR

Comment (orangejulius) [reply utils/download_sqlite_all.js]: Makes sense. I just put in #486 to fix line 56, which had a similar error. Want to open another PR? ;)

Comment (loadit1) [reply utils/download_sqlite_all.js]: Sure, updated in #484

**Pull request merged**